tests/Ltest-mem-validate.c

```diff
@@ -108,6 +114,9 @@ main (int argc, char **argv UNUSED)
 
   stack_start = &start;
 
+  char string[4096];
```

Comment:
I'd suggest using PAGE_SIZE here to match the code above.  And maybe a comment mentioning that we need to allocate as much as is being protected by the mprotect call.